### ISSUE TITLE
Undefined constant TopShelfCraft\Wordsmith\libs\Emoji::KEYCAP

### ISSUE THREAD

**Issue by mattbloomfield** [TopShelfCraft/Wordsmith#55]
I'm trying to upgrade a site to Craft4 and apparently it's using Wordsmith's `emojify` filter. I don't know much about it, but this is the template use: 

```
<div class="moderator__content">
            <h4>{{ entry.mainHeading|raw }} {{ entry.emoji|emojify }}</h4>
            <p>{{ entry.description }}</p>
            <br><br>
</div>
```

I'm on Craft Pro 4.5.11 using Wordsmith 4.2.0

Here is the error I'm getting: 

![image](https://github.com/TopShelfCraft/Wordsmith/assets/37582550/0086f637-56ad-46be-982b-f0b675542aee)

**Comment by BenParizek** [comment]
Some additional info in this duplicate, ticket which I just closed.

https://github.com/TopShelfCraft/Wordsmith/issues/56

**Comment by michaelrog** [comment]
Fixed in dd220fe, released in `4.3.0`.

**Comment by mattbloomfield** [comment]
Thanks Mike!
